GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/Main.java

```diff
@@ -10,21 +10,14 @@
 
     public static void main(String[] args) throws IOException {
 
-//        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
-//
-//        String[] firstMultipleInput = bufferedReader.readLine().replaceAll("\\s+$", "").split(" ");
-//
-//        int m = Integer.parseInt(firstMultipleInput[0]);
-//
-//        int n = Integer.parseInt(firstMultipleInput[1]);
-//
-//        List<String> magazine = Stream.of(bufferedReader.readLine().replaceAll("\\s+$", "").split(" ")).collect(toList());
-//
-//        List<String> note = Stream.of(bufferedReader.readLine().replaceAll("\\s+$", "").split(" "))
-//                .collect(toList());
-//
-//        Result.checkMagazine(magazine, note);
-//
-//        bufferedReader.close();
-//    }
-}}
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
+        String[] firstMultipleInput = bufferedReader.readLine().replaceAll("\\s+$", "").split(" ");
+
+        int m = Integer.parseInt(firstMultipleInput[0]);
```

Comment:
m? I know you don't have much time, but try not to make the same mistakes that we already reviewed